formula-test/src/test/java/com/instacart/formula/test/TestFormulaTest.kt

```diff
@@ -73,7 +73,9 @@ class TestFormulaTest {
         class Button(val onNameChanged: (String) -> Unit)
     }
 
-    class FakeChildFormula : TestFormula<ChildFormula.Input, ChildFormula.Button>(), ChildFormula {
```

Comment:
Using `testFormula` in combination with `implementation` allows us to avoid re-typing various types that are already declared within `ChildFormula`